.travis.yml

```diff
@@ -1,3 +1,50 @@
-language: shell
-script:
-  - bash -c 'shopt -s globstar; shellcheck **/*.sh'
+language: minimal  # Use minimalistic image just with docker and basic tools.
+
+services:
+  - docker
```

Comment:
why do you use docker here?

I know nothing about Travis, but the docs seem to me that docker is not necessary for what you do.

https://docs.travis-ci.com/user/docker/

https://docs.travis-ci.com/user/tutorial/

Reply:
Argh, I take it back, I didn't notice the docker commands

Reply:
By default there is no podman and get it installed on ubuntu is not straightforward. Docker needs a system service that needs to be started. Docker is necessary to start Centos on top of ubuntu.